src/core/data-tests.js

```diff
@@ -23,11 +23,42 @@ export const name = "core/data-tests";
 const lang = defaultLang in l10n ? defaultLang : "en";
 
 function toListItem(href) {
-  return hyperHTML.bind(document.createElement("li"))`
+  
+  var [testFile] = new URL(href).pathname.split("/").reverse();
```

Comment:
ok, call this something else and don't reassign it... otherwise, its confusing to know what this actually is.